static/coffee/flows/controllers.coffee

```diff
@@ -655,7 +655,7 @@ app.controller 'FlowController', [ '$scope', '$rootScope', '$timeout', '$log', '
           DragHelper.showSendReply($('#' + category.sources[0] + ' .source'))
         ,0
 
-  $scope.addAction = (actionset) ->
+  $scope.addAction = (actionset, innerAction=null) ->
 
```

Comment:
Not sure what innerAction could mean for addAction either. Can you help me understand why this is needed?

Reply:
Maybe the nomenclature has caused confusion, `innerAction` could be a conditional like `isNewStepAction` or something else (when the plus button is tapped on flow step). I needed this argument to control when it needs to restart the quick replies variables in `NodeEditorController`.